src/compiler/output-targets/dist-lazy/generate-system.ts

```diff
@@ -61,7 +61,7 @@ const getSystemLoader = async (config: d.Config, compilerCtx: d.CompilerCtx, cor
 
     var resourcesUrl = scriptElm ? scriptElm.getAttribute('data-resources-url') || scriptElm.src : '';
     var start = function() {
-      var url = new URL('${corePath}', resourcesUrl);
+      var url = new URL('${corePath}', window.location.href + resourcesUrl);
```

Comment:
window.location.origin would be better because if for some reason there was a hash fragment in the URL it would break this.
`
var url = new URL('${corePath}', new URL(resourcesUrl, window.location.origin)) would be more reliable IMO
`

Reply:
Good point! 

A hash fragment would actually be dropped by `new URL`, because it takes the second parameter as the base path. But for handling subpages in an SPA it should definitely point to the origin.

So it should be 
`var url = new URL('${corePath}', new URL(resourcesUrl, window.location.origin))`
for sure. I''ll update